slsDetectorSoftware/src/slsDetector.cpp

```diff
@@ -2666,6 +2677,143 @@ uint32_t slsDetector::getADCEnableMask() {
     return shm()->adcEnableMask;
 }
 
+void slsDetector::setADCInvert(uint32_t value) {
+    uint32_t arg = value;
+    FILE_LOG(logDEBUG1) << "Setting ADC Invert to 0x" << std::hex << arg << std::dec;
+    if (shm()->onlineFlag == ONLINE_FLAG) {
+        sendToDetector(F_SET_ADC_INVERT, &arg, sizeof(arg), nullptr, 0);
```

Comment:
You can  use sendToDetector without specifying the sizes
e.g. sendToDetector(F_SET_ADC_INVERT, arg, nullptr)